mopidy_party/__init__.py

```diff
@@ -31,7 +29,7 @@ def get(self):
         else: # Valid vote
             self.data["votes"].append(self.request.remote_ip)
             if (len(self.data["votes"]) == self.requiredVotes):
-                self.core.playback.next()
+                next(self.core.playback)
```

Comment:
This change is incorrect and should be reverted. `self.core.playback` is not an iterator where `.next()` returns the next value. It's the playback controller, and `next()` is a regular method that changes to the next track.

Reply:
fixed, thanks.